sklearn/neighbors/_kde.py

```diff
@@ -36,8 +37,10 @@ class KernelDensity(BaseEstimator):
 
     Parameters
     ----------
-    bandwidth : float, default=1.0
-        The bandwidth of the kernel.
+    bandwidth : float or {"scott", "silverman"}, default=1.0
```

Comment:
I would also add a bit of documentation in the User Guide: https://scikit-learn.org/stable/modules/density.html#kernel-density

Just after the paragraph:

```
The bandwidth here acts as a smoothing parameter, controlling the tradeoff between bias and variance in the result. A large bandwidth leads to a very smooth (i.e. high-bias) density distribution. A small bandwidth leads to an unsmooth (i.e. high-variance) density distribution.
```

We can add a paragraph:

```
The parameter `bandwidth` controls this smoothing. One can either set
manually this parameter or use either the Scott's and Silvermann's estimation
methods.
```